users/vieting/experiments/switchboard/hybrid/feat/specaug_random.py

```diff
@@ -0,0 +1,152 @@
+def _random_mask_mixed(x, batch_axis, axis, amount):
```

Comment:
Wouldn't it be easier to just introduce a flag in `_mask()` that allows shuffling the mask (`cond`)? Then you could also keep the same randomness w.r.t. the number of masked channels.

Reply:
most of the code in _mask is designed to work with fixed masks and their indices. I figured it would make more sense to make a separate function since the differences in code are so big.

Reply:
The only argument against it would be performance and code clarity. But I agree that it would reduce the chance of potential new bugs or differences. I will change this and restart the trainings with the new code.